packages/cache-core/src/cacheHandler.ts

```diff
@@ -194,14 +194,14 @@ export class Cache implements CacheHandler {
     }
   }
 
-  static addCookie(value: string) {
-    Cache.cacheCookies.push(value)
+  static addCookie(value: string | string[]) {
+    Cache.cacheCookies.push(...value)
```

Comment:
if user passed just a string it is going to create rule for each character, like: `...'abc' -> ['a', 'b', 'c']`

let's create just a separate method to accept arrays OR have just 1 method which accepts array

Reply:
fixed